pkg/networkservice/common/serialize/client.go

```diff
@@ -25,10 +25,11 @@ import (
 	"github.com/networkservicemesh/api/pkg/api/networkservice"
 
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
+	"github.com/networkservicemesh/sdk/pkg/tools/multiexecutor"
 )
 
 type serializeClient struct {
-	executor multiExecutor
+	executor multiexecutor.MultiExecutor
```

Comment:
This feels like a separate PR to introduce multiexecutor into serialize.  Could we split it out?